src/clj/harja/palvelin/integraatiot/api/varusteet.clj

```diff
@@ -174,17 +169,17 @@
   (let [livitunniste (livitunnisteet/hae-seuraava-livitunniste db)
         toimenpiteen-tiedot (:varusteen-lisays data)
         tietolaji (get-in toimenpiteen-tiedot [:varuste :tietue :tietolaji :tunniste])
-        tietolajin-arvot (get-in toimenpiteen-tiedot [:varuste :tietue :tietolaji :arvot])
+        tietolajin-arvot (assoc (get-in toimenpiteen-tiedot [:varuste :tietue :tietolaji :arvot]) :tunniste livitunniste)
```

Comment:
Harja hallitsee sen kautta lisättyjä tunnisteita. Näin ollen Harjan avaruudesta generoitu tunniste. pitää erikseen lisätä arvoihin käsin.